crypto/rsa/rsa_gen.c

```diff
@@ -145,8 +145,10 @@ int ossl_rsa_multiprime_derive(RSA *rsa, int bits, int primes,
             tmp = BN_dup(r1);
             if (tmp == NULL)
                 goto err;
-            if (!sk_BIGNUM_insert(pplist, tmp, sk_BIGNUM_num(pplist)))
+            if (!sk_BIGNUM_insert(pplist, tmp, sk_BIGNUM_num(pplist))) {
+                BN_free(tmp);
                 goto err;
+            }
```

Comment:
Why do the free locally here for `tmp` but for `newexp` and `dval` put them into the cleanup list at `err:`? Wouldn't it be more consistent to add a `tmp = NULL;` before the break here and at (old) line 161?

Reply:
Because the style was already inconsistent; and I thought that since we only need to do the clean up action once in this case we can do it directly here. For the other cases we would need to insert the same code multiple times if we wanted to do it directly.

Reply:
> since we only need to do the clean up action once in this case we can do it directly here

... and since it actually needed to be done twice, are you going to change it? I will approve if you really don't want to change it, but I'm generally a fan of reducing inconsistency...